fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -1436,15 +1420,20 @@ public void write(DataOutput out) throws IOException {
         out.writeLong(pauseTimestamp);
         out.writeLong(endTimestamp);
 
+        /*
```

Comment:
Remove these comment

Reply:
Finished. And I add some comments to explain the indicators called `errorRowsAfterResumed`.